src/test/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommandTest.java

```diff
@@ -76,9 +78,31 @@ public void execute_duplicatePendingOrder_warningGiven() {
         assertCommandSuccess(addOrderCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_similarPendingOrder_warningGiven() {
+        // add something similar(but not identical) to the first order to the person's order list
+        String editedItemString = normalise(ABACUS.getItem().fullDescription);
+        Order toAdd = new OrderBuilder(ABACUS).withItem(editedItemString).build();
+        AddOrderCommand addOrderCommand = new AddOrderCommand(INDEX_FIRST, toAdd);
+
+        //ensures it is not equal to ABACUS
+        assertNotEquals(toAdd, ABACUS);
+
+        model.getFilteredPersonList().get(0).getOrderList().add(toAdd.createCopy());
```

Comment:
Shouldn't this be adding `ABACUS` instead? From what i'm seeing, it seems you're adding a copy of `toAdd` first and then executing the addOrderCommand to add another `toAdd`, which is the same as `execute_duplicatePendingOrder_warningGiven()`

Reply:
in